src/ast/mod.rs

```diff
@@ -3831,19 +3831,19 @@ impl fmt::Display for Statement {
                         .map(|to| format!(" TO {to}"))
                         .unwrap_or_default()
                 )?;
+                if !columns.is_empty() {
```

Comment:
could you explain what this change does? it looks like its swapping the order in which things show up but then I figure that should've led to some test failing, so wondering if I'm misunderstanding the effect of the change

Reply:
`COMMENT` of view should be after a column list for example in

1.  [Snowflake](https://docs.snowflake.com/en/sql-reference/sql/create-view) 
2. [BigQuery](https://cloud.google.com/bigquery/docs/reference/standard-sql/data-definition-language#create_view_statement)
3. [Databricks](https://docs.databricks.com/en/sql/language-manual/sql-ref-syntax-ddl-create-view.html)

I found that an order is invalid with `Display` while testing current changes with a query from production.

This fix makes order of displaying `COMMENT` options aligning sql specification.

Reply:
I added a dedicated unit-test for Snowflake to cover this case.

Reply:
I see, It doesn't seem like BigQuery supports COMMENT so should be unaffected. For this part, what's the intention behind the change?
```rust
if matches!(options, CreateTableOptions::With(_)) {
                    write!(f, " {options}")?;
}
```

Reply:
An order of entities if CREATE VIEW in `Snowflake` and `Databricks` cases DDL

`CREATE VIEW` -> `COLUMN LIST` -> [`(WITH) OPTIONS`] -> [`COMMENT`]

This change puts all available entities in order aligned DDL specifications. An order in main is reversed for column list, [with] options and comment.

Reply:
Ah, so the change of `CreateTableOptions::With(_)` I think is lacking test coverage in the MR, which made it hard to see why it was needed. Can we add tests for that as well?

Reply:
@7phs I think just one follow up needed on this one to add a test demonstrating the WITH behavior? i.e `CREATE OR REPLACE VIEW v (a COMMENT 'a comment', b, c COMMENT 'c comment') WITH (foo=bar) COMMENT = 'Comment' AS SELECT a FROM t`

Reply:
@iffyio I added this case (as one more case) into the existing unit test.